### ISSUE TITLE
Flashing "Applied force" and "Friction information" FF only

### ISSUE THREAD

**Issue by orejolaphet** [phetsims/forces-and-motion-basics#107]
Slow performance if you rapidly click an one single object on the track for the "Motion" "Acceleration" and "Friction" screens. The Applied force and Friction information flash between being highlighted and not, rapidly after being clicked rapidly. 

OSX 10.9.4
FF 33.1.1
To reproduce:
-Open sim
-Select any of the scenes with tracks (Acceleration, Motion, Friction)
-Place a single object on the track
-Rapidly click the object

Troubleshooting information (do not edit):
Name: Forces and Motion: Basics
URL: http://www.colorado.edu/physics/phet/dev/html/forces-and-motion-basics/1.1.0-rc.1/forces-and-motion-basics_en.html
Version: 1.1.0-rc.1
Features missing: touch
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.9; rv:33.0) Gecko/20100101 Firefox/33.0
Language: en-US
Window: 1280x646
Pixel Ratio: 1/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (Mozilla)
Vertex: attribs: 16 varying: 16 uniform: 1024
Texture: size: 4096 imageUnits: 16 (vertex: 16, combined: 16)
Max viewport: 8192x8192
OES_texture_float: true

**Comment by jessegreenberg** [comment]
Performance issues likely related to #105.  It looks like the 'Applied Force' and 'Friction' slider labels are being grayed out when the force controls are disabled.

**Comment by jessegreenberg** [comment]
Looking through MotionControlPanel and MotionScreenView, changing the text fill to gray when disabled is an intentional behavior.  This makes sense for the 'Applied Force' information and controls, but I don't understand why this is done for the 'Friction' slider tick labels.

**Comment by jessegreenberg** [comment]
```js
    var createFrictionSlider = function() {
      var createTick = function( label, visible ) {
        var path = new Path( Shape.lineSegment( new Vector2( 0, 0 ), new Vector2( 0, -18 ) ), {
          stroke: 'black',
          lineWidth: 1
        } );
        var text = new Text( label, { font: new PhetFont( 15 ) } );
        model.stack.lengthProperty.link( function( length ) {
          var enabled = length > 0;
          path.fill = enabled ? 'black' : 'gray';
          text.fill = enabled ? 'black' : 'gray';
        } );
        return new VBox( { children: [ text, path ], pickable: false, visible: visible } );
      };
```

**Comment by jessegreenberg** [comment]
@ariel-phet is this the desired behavior?  Are the friction slider labels supposed to be grayed out when there are no items to push?

![ezgif com-video-to-gif](https://cloud.githubusercontent.com/assets/6396244/13024145/2823f58e-d1ae-11e5-8b45-cc98c910525f.gif)

**Comment by ariel-phet** [comment]
@jessegreenberg that behavior definitely does not seem to be desired. I am guessing that when this sim was released since we did not have a solidified QA process this was missed.

I am not sure why this was implemented in the code...the friction slider is still active when no objects are present, and you are adjusting the coefficient of friction. I guess if you are getting really technical, a coefficient of friction only really exists between two surfaces (rubber and wood, or such). But that seems like a nuance we do not need to highlight, and the model assume an equivalent coefficient for all objects in the sim.

So no, the behavior is not desired, the friction slider should not change in appearance.
